live-build/misc/upgrade-scripts/verify-impl

```diff
@@ -26,6 +26,25 @@ function usage() {
 	exit 2
 }
 
+function get_container_name() {
+	return basename $(df -k / | tail -1 | awk '{print $1}')
```

Comment:
how about changing this to:
```
basename $(zfs list -Ho name /)
```
?

I've also never seen `return` used to return a string, but I'm not a bash expert. Normally I'd do something like:
```
basename $(zfs list -Ho name /)
```
(i.e. no `return`).. and then it'd be used like this:
```
mkdir -p "${UPDATE_VERIFY_LOG_DIR}/$(get_container_name)"
```
i.e. using `$(...)` instead of `${...}`.

I'm fine if it works your way; I'll have to read up on the docs since I might have just learned something new :)

Reply:
Sorry about that. Review didn't get update with latest change. `return` doesn't work with string like you said. Should've been `echo`. However, I'm going with your suggestion of exposing environment variable from `upgrade-container` level, so this function will get removed on the next revision.